### ISSUE TITLE
v1.0.0 (latest)

### ISSUE THREAD

**Issue by serapath** [alincode/digital-nomad-guide-to-taiwan#1]
Hey :-)

You are really experienced already.
I wonder what is your experience to work remote.

### how we collaborate

For example - in ethereum - the way we work with them is to make an Issue `vX.Y.Z (next)`,
* which contains a bullet point list of all the issue that need to be finished to release `vX.Y.Z (next)`
* each bullet point item has a title and tweet sized summary and link to another issue with the details
* this leads to a "tree structured" issue list (because subissues can have more subissues)
* we assign the issues to collaborators and when we work on them open and link "work in progress" PULL REQUESTS to the issues ...which we code review and close and check mark the done issue
* when all bullet point issues are finished.
* we rename the issue to `vX.Y.Z (latest)` and close the issue and release the version `vX.Y.Z`

... if we have some more ideas while we work on `vX.Y.Z (next)` we put them into `vXXX (future)`
* (basically the version issue that will become the next `vXXX (next)` after we finish the current `vX.Y.Z`

...we follow https://semver.org/ with our versioning.

### collaborators for `digital-nomad-guite-to-taiwan`

It would be super cool if you could make some beginner issues for people who are learners of javascript and css and html to practice. You could "practice" to outsource work and mentor beginners who in return solve some simple tasks. ...maybe you can even make your project a bit bigger and more ambitious and find some other experts that could make "sub apps" that you can integrate as iframes ...or they could produce "modules" that you could `import/require` in your project.

Maybe you can even find people who are interested in `smart contracts` and brainstorm about ideas how to integrate some `smart contract p2p backend` to communicate with via API :-)

That way you might also learn something new and be able to make an even better app, because you don't have to do everything yourself :-)

what do you think? ;-)

---

**If you want I can even show you an example personally how we work in ethereum**

If you feel like - I would also be very interested to learn how you work remote with others

**Comment by alincode** [comment]
@serapath 

Most of time, I am a javascript backend developer, sometime I am a project manager.
I will done basic website system design, and split task to small issues, then other young programer could easy understand how to do.
I like coding and travel, so I quit job, be a digital nomad.

**Comment by alincode** [comment]
@serapath 

0.1.0 maybe add more image and information data
0.2.0 maybe add google map api
0.3.0 I still thinking...hahaha

**Comment by serapath** [comment]
Here is the example of how we structure our work in ethereum
https://github.com/ethereum/browser-solidity/issues/791

**Comment by serapath** [comment]
Maybe you can make some issues here on your projects to describe and plan your project work for the weekend and post it on the chat. Hopefully some people feel like trying to collaborate and join the project idea :-)

**Comment by alincode** [comment]
@serapath 

I was JAVA web developer, but JAVA usually in the big project, then big project that meaning, the company can't allow work remotely. so I change my language started 2 years ago.

* My first remote job is Automation Engineer, because they could let me work remotely. : )
* 2th remote job is javascript developer, work in my friend's company, we got some customized website (most is e-commerce website) project. after half year, team merger with other company.
* 3th remote job, I be a freelancer, I wrote web crawler to collection some data for someone.
* 4th remote job, I work in my other friend's company. doing some customized website project again.

**Comment by serapath** [comment]
Why would the company of a big project not allow remote work?
This sounds to me like an excuse managers  make who are incapable of organizing remote work.
It's more cost efficient.

Sounds pretty cool otherwise.

**Comment by alincode** [comment]
@serapath 
They have security issue consider, e.g. bank system, factory system, telecommunication system.
some company said, "yes, we agree work remotely is good for someone, but for other people not, we need to consider all of employees, if we make a exception for you, that will break the company culture",
sure, I think that is excuse too.
they just want to make a rules for anyone, so they can easy manager employee.

**Comment by serapath** [comment]
yes - but in the US - most companies started to prepare for the remote work era.
Already > 35% of the population is working remote and it's growing fast.

Security is just a question of having tasks prepared in a way and systems set up in ways that they can handle remote work. But I agree - traditionally companies are not prepared - so yes, of course - they need time to adapt. European companies for example have the same excuses - it just means they will need a few years more time.

In the mean time the best people will start working remote for high salaries in the US or other english speaking countries until their home countries catch up :-) ....but maybe remote working people become self employed and outsource more work to their self employed partners in order to not make traditional companies necessary anymore.

I would prefer that anyway :P
